### ISSUE TITLE
The program freezes

### ISSUE THREAD

**Issue by YourName2024** [MaterialDesignInXAML/MaterialDesignInXamlToolkit#3448]
### Bug explanation

Hello. I decided to write a program using your styles. I connected them to resources and started using them. As a result, I noticed when the computer is blocked and we log in. The program freezes completely and the buttons and elements do not work. Interestingly, the double-click action is hung on the DataGrid. Which brings up the edit button. And this action works. But it opens a completely empty form.

### Version

1.2.7 and 3.2.0

**Comment by YourName2024** [comment]
I solved this problem, but I don't know how much is correct. I didn't use hide() on the first form, but Close()

**Comment by Keboo** [comment]
@YourName2024 are you able to provide an example that reproduces the problem. There is not enough information here to provide a fix or solution.

**Comment by jf1187gh** [comment]
> @YourName2024 are you able to provide an example that reproduces the problem. There is not enough information here to provide a fix or solution.

I'm having the same issue. After opening a new window and hiding the main window, if the user locks/unlocks the screen, then the application locks up. 

My current work around is to disable hardware acceleration (RenderOptions.ProcessRenderMode = RenderMode.SoftwareOnly;)

-WPF, .net 8, windows application

Code calling new new window:
CheckListForm cLWin = new()
{
    Owner = Application.Current.MainWindow,
    WindowStartupLocation = WindowStartupLocation.CenterOwner,
    MainWindowReference = this
};
cLWin.Show();
Hide();
![Lockup](https://github.com/MaterialDesignInXAML/MaterialDesignInXamlToolkit/assets/97968929/05583128-ef81-4be2-853b-4b1360b09d54)

**Comment by Keboo** [comment]
@jf1187gh are you able to provide a sample repo that reproduces the issue?

**Comment by jf1187gh** [comment]
Were you able to reproduce the issue with my sample code?

**Comment by MichelMichels** [comment]
This will be fixed with #3616 . 

@jf1187gh the solution to your problem is adding following inside your `Window` tag: 

```xaml
<Window
    xmlns:md="http://materialdesigninxaml.net/winfx/xaml/themes"
    md:ShadowAssist.CacheMode="{x:Null}" 
```
